From 21120032d414f435bc23b74ea25ffc8b5e0b95ba Mon Sep 17 00:00:00 2001
From: Vishalcj17 <vishalcj@aospa.co>
Date: Tue, 27 Feb 2024 18:23:01 +0900
Subject: [PATCH 21/34] cmds: Completely kill bootanimation resizing.

02-27 14:46:25.818  1781  1909 V BootAnimation: Resizing the boot animation surface to 1440 3216

looks ugly and out-of-place when resized. Only thing which might be compatible with this is the "android" logo bootanimation.

Change-Id: Ic22be3c002481d62bbee1ce46d76e54c65295fad
---
 cmds/bootanimation/BootAnimation.cpp | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/cmds/bootanimation/BootAnimation.cpp b/cmds/bootanimation/BootAnimation.cpp
index b43905b19..973c96f3a 100644
--- a/cmds/bootanimation/BootAnimation.cpp
+++ b/cmds/bootanimation/BootAnimation.cpp
@@ -434,6 +434,7 @@ public:
             return 1; // keep the callback
         }
 
+#if 0
         constexpr int kBufferSize = 100;
         DisplayEventReceiver::Event buffer[kBufferSize];
         ssize_t numEvents;
@@ -469,6 +470,7 @@ public:
                 }
             }
         } while (numEvents > 0);
+#endif
 
         return 1;  // keep the callback
     }
-- 
2.50.1

